core/Symbols.cc

```diff
@@ -476,18 +558,18 @@ SymbolRef Symbol::findMemberTransitiveInternal(const GlobalState &gs, NameRef na
     if (maxDepth == 0) {
         if (auto e = gs.beginError(Loc::none(), errors::Internal::InternalError)) {
             e.setHeader("findMemberTransitive hit a loop while resolving `{}` in `{}`. Parents are: ", name.show(gs),
-                        this->showFullName(gs));
+                        ref(gs).showFullName(gs));
```

Comment:
Agreed. Getting rid of `ref` is on my TODO once we can specialize symbol classes. There are a couple of ways we can fix this:

* Move string methods back down to specialized Symbol classes once we have them.
* Move this function to *ref classes so that `this` is `SymbolRef`.